packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -70,9 +71,57 @@ $button-width: 124px;
   }
 }
 
-.button {
+.buttonPrimary {
+  &:hover {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &:not(.disabled):not(:disabled) {
```

Comment:
With the css tree being a duplicate from above, developers will need to make sure that both `.buttonDefault` and `.buttonPrimary` are kept in sync. I'm thinking a mixin could be used here to solve this issue.

Reply:
created a `mixin` for duplicate code. Nor sure if this what you are pointing out or if there is a better way to do it

Reply:
Cool! I was also thinking of including parameters in the mixin. So something like:

```scss
.buttonPrimary {
  @include button-mixin(
    $background-color: add-tint($ca-palette-seedling, 10%),
    $border-color: $ca-palette-seedling,
    $active-background-color: $ca-palette-seedling,
    $disabled-background-color: add-tint($ca-palette-ink, 90%),
    $disabled-border-color: add-tint($ca-palette-ink, 90%)
 )
}
```

That way we don't need to synchronise the css tree structure between variants.

Reply:
Nice, to be honest didn't know about that. Just updated the PR to include colors! Thanks @jeremy8883